ext/standard/config.w32

```diff
@@ -1,6 +1,18 @@
 // vim:ft=javascript
 // $Id$
 
+ARG_WITH("argon2", "Argon2 support", "no");
+
+if (PHP_ARGON2 != "no") {
+	if (CHECK_LIB("Argon2Ref.lib", "argon2", PHP_ARGON2)
+	&& CHECK_HEADER_ADD_INCLUDE("argon2.h", "CFLAGS_ARGON2")) {
+		AC_DEFINE('HAVE_ARGON2LIB', 1);
+		EXTENSION("argon2", null, false);
```

Comment:
This doesn't look right, simply drop the call to EXTENSION() here and leave the AC_DEFINE().